blog/2020-03-01-CombiningBPMNWithOptaplanner.adoc

```diff
@@ -0,0 +1,611 @@
+= Combining BPMN with Optaplanner
+:page-interpolate: true
+:awestruct-author: Christopher-Chianelli
+:awestruct-layout: blogPostBase
+:awestruct-tags: [useCase]
+:awestruct-date: 2020-03-01
```

Comment:
Remove this line, it parses the date automatically from the filename.
Rename the filename to next tuesday